pyproject.toml

```diff
@@ -35,6 +35,7 @@ dependencies = [
     "marshmallow-enum",
     "marshmallow-jsonschema>=0.12.0",
     "mashumaro>=3.11",
+    "msgpack>=1.1.0",
```

Comment:
nit: do we have to be so strict? 1.1.0 was released on [9/10/24](https://pypi.org/project/msgpack/#history).

Reply:
No we don't need, but the newer it is, the it can fix more bug in msgpack